chatui/src/app/modules/home/create-agent-dialog.tsx

```diff
@@ -53,19 +108,52 @@ const CreateAgentDialog = (props: { open: boolean; onOpenChange: (open: boolean)
 							<Label htmlFor="human" className="text-right">
 								Human
 							</Label>
-							<Input id="human" name="human" defaultValue="cs_phd" className="col-span-3" />
+							<DropdownMenu>
```

Comment:
I actually think the select primitive is what we want here: https://ui.shadcn.com/docs/components/select

What do you think?